extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -21,6 +21,15 @@ export class AmpCarouscroll extends BaseCarousel {
   }
   /** @override */
   buildCarousel() {
+    this.element.classList.add('-amp-slidescroll');
+
+    /** @private {!Array<!Element>} */
+    this.slides_ = this.getRealChildren();
+
+    this.slides_.forEach(slide => {
```

Comment:
Let me add a todo there once i have everything set and working then we can try and swap it in. ill add the container in for now.